src/components/marketplace/order_book.tsx

```diff
@@ -48,10 +48,13 @@ const orderToRow = (
     }, new BigNumber(0));
 
     const mySizeConverted = tokenAmountInUnits(mySize, baseToken.decimals, UI_DECIMALS_DISPLAYED_ORDER_SIZE);
+    const zeroDecimals = '0'.repeat(UI_DECIMALS_DISPLAYED_ORDER_SIZE);
+    const mySizeStrDisplay = '0.'.concat(zeroDecimals);
+    const displayColor = mySizeConverted !== mySizeStrDisplay ? undefined : '#dedede';
     const mySizeRow =
         web3State !== Web3State.Locked && web3State !== Web3State.NotInstalled ? (
-            <CustomTD styles={{ tabular: true, textAlign: 'right' }}>
-                {mySizeConverted !== '0.00' ? mySizeConverted : '-'}
+            <CustomTD styles={{ tabular: true, textAlign: 'right', color: displayColor }}>
+                {mySizeConverted !== mySizeStrDisplay ? mySizeConverted : '-'}
```

Comment:
Q: Instead of this, can we do something like `!mySize.eq(new BigNumber(0))`?

Reply:
Yes, it's a good idea, It's done now